src/main.ts

```diff
@@ -692,7 +704,7 @@ export default class WhatsApp {
             this.setupEncryptionKeys(data as WhatsAppConnPayload);
             setTimeout(this.keepAlive.bind(this), 20 * 1000);
 
-            if (this.keysPath) {
+            if (this.parameters.keysPath) {
```

Comment:
```suggestion
            if (this.opts.keysPath) {
```
Update property again